css/style.css

```diff
@@ -1,19 +1,298 @@
+}
+
+li{
+    font-weight: normal;
+    font-size: 22px;
+}
+
+p{
+    font-size: 33px;
+    color:var(--black)
+}
+a:link,:visited,:active+.btn-store {
+    color: white;
+    text-align: center;
+}
+/* //General Style */
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    background-color: white;
+}
+
+nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+      position: fixed;
+    width: 100%;
+    height: 90px;
+    padding: 0 120px;
+    background:hsla(0, 0%, 98%, 0.954);
+}
+
+hero{
+    background-color: rgb(199, 84, 21);
+}
+footer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+footer h5{
+margin-bottom: 30px;
+
+}
+button{
+    width: 199px;
+    height: 68px;
+    font-size: 22px;
+    border-radius: 5px;
+    border:none;
+    background: #E0633A;
+}
+
+
+/* //Logo */
+.logo{
+
+    height: 50px;
+    margin-right: auto;
+}
+
+/* //NAV */
+.nav__links, li, a{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    height: 100%
+}
+
+.nav__links a{
+    color:var(--grey-light);
+}
+nav .nav__link_on{
+    font-style: normal;
+    font-weight:var(--fw-500);
+    color:var(--grey-dark)
+}
+.nav__links a:hover{
+    color:var(--orange);
+    
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.nav__links li{
+    padding-left: 40px;
+}
+
+/* //Hero */
+.hero{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 852px;
+    background-repeat: no-repeat;
+    background-image:url(../img/first-background.jpg);
+    background-size:cover;
+    background-position-x: center;
+}
+
+.hero-intro{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 130px;
+}
+
+.hero-intro, button{
+    margin-top: 70px;
+    color:var(--white)
+}
+
+/* //Features */
+.features{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 170px 0;
+}
+
+.feature-flex-parent {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
 
+.feature-flex-item{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    padding: 78px 0 0;
+    width: 400px;
+}
+
+.feature-flex-item img{
+    height: 156px;    
+}
+.feature-flex-item p{
+    padding-top: 30px;    
+}
+
+ /* Quotes */
+.quotes{
+    display: grid;
+    grid-template-columns: 4.268fr 8.8fr;
+    height: 439px;
+}
+
+.quote-photo{
+
+    background-color: #E0633A;
+    background-image:url(../img/feature-img.png);
```

Comment:
remember background-image CSS property should only be used for true background images. If the image is part of the layout, it should be inserted in the HTML